plugins/module/util/auth.py

```diff
@@ -0,0 +1,17 @@
+from keycloak import KeycloakOpenID
+from os import environ;
+
+def get_access_token():
+    offlineToken = environ.get('OFFLINE_TOKEN')
```

Comment:
In architecture we need to decide if authentication is defined in ansible as secret or we use env vars.